modules/quickstart/pages/quick_start-rest.adoc

```diff
@@ -121,7 +121,7 @@ To check if a keyspace or tables exist, execute the REST query with `cURL`:
 
 [source,bash]
 ----
-curl -L -X GET 'localhost:8082/v1/keyspaces' \
+curl -L -X GET 'localhost:8082/v2/keyspaces' \
```

Comment:
This should either stay as v1 or change to

```
╰─$ curl -L -X GET 'localhost:8082/v2/schemas/keyspaces' \
--header 'accept: application/json' \
--header 'content-type: application/json' \
--header 'X-Cassandra-Token: $AUTH_TOKEN'
{"data":[{"name":"system_distributed"},{"name":"system"},{"name":"data_endpoint_auth"},{"name":"system_schema"},{"name":"stargate_system"},{"name":"system_auth"},{"name":"system_traces"}]}
```

since just `/v2/keyspaces` doesn't exist

Reply:
The response will also have to change since v1 returns a list of strings and v2 returns a list of objects.